run_config.json

```diff
@@ -1,113 +1,4 @@
-[
```

Comment:
Issue: `run_config.json` should be changing in this PR

Reply:
I'll remove this file.